suzieq/engines/pandas/device.py

```diff
@@ -77,7 +80,7 @@ def get(self, **kwargs):
                 .reset_index(drop=True)
 
             df = df.merge(poller_df, on=['namespace', 'hostname'],
-                          how='outer', suffixes=['', '_y'])  \
+                          how='outer', suffixes=['', '_y'], indicator=True) \
```

Comment:
I think you added the indicator only for debug. It's not used in the next code lines. We should remove it.

Since you are working on these lines, can you fix the pyright problem as follows?
``` suggestion
                          how='outer', suffixes=('', '_y')) \
```

Reply:
Done